### PR TITLE
Prevents admins messing with eachothers votes.

### PR DESCRIPTION
**Pull request by Goldsteel**
[administration][QoL][featurerequest]

Dorms reported that admins could monkey with eachothers votes. This is probably because we use a global variable and if both admins are in the loops of a proc they can write stuff to the same area. Not anymore, though. If you try and push buttons after someone else has before the vote finishes or is cancelled, everyone in the admin list is alerted that you tried to start another vote.

:cl:
 * bugfix: Adds sanity to the voting proc to prevent two admins editting the details at once by accident.

### CODE REVIEW

**Comment by SysadminJeroen** [comment]
We had 11 admins online yesterday and a few making a vote (because the mapvote fucked up the round before) which bugged eachother's vote out.

Having a lock-out would be nice so we don't get a "too many cooks in the kitchen" situation.

**Comment by Goldsteel** [comment]
This currently works on a first come first serve basis. It will unlock only if a vote ends or the vote is cancelled by someone in the voting panel. I may add WHO cancelled the vote, too.

**Comment by SonixApache** [comment]
for future ref, delaying and then forcing another mapvote (with a proc call) is a good way to unfuck it

File: code/modules/html_interface/voting/voting.dm [hunk 1]
```diff
@@ -271,6 +275,11 @@ var/global/datum/controller/vote/vote = new()
 	return 0
 
 /datum/controller/vote/proc/initiate_vote(var/vote_type, var/initiator_key, var/popup = 0, var/weighted_vote = 0)
+	if(currently_voting)
+		for(var/client/C in admins)
+			to_chat(C, "<span class='info'>[C.ckey] attempted to begin a vote, however a vote is already in progress.</span>")
```
Comment (Kurfursten) [inline code/modules/html_interface/voting/voting.dm]: use message_admins()

Comment (Goldsteel) [reply code/modules/html_interface/voting/voting.dm]: ill do it when i get back

Comment (Goldsteel) [reply code/modules/html_interface/voting/voting.dm]: it does "ADMIN LOG: Goldsteel attempted to begin a vote, however a vote is already in progress." now and seems fine

**Pull request merged**